src/main/java/org/jitsi/jigasi/transcription/WhisperTranscriptionService.java

```diff
@@ -174,8 +195,7 @@ public void addTranscriptionListener(TranscriptionListener listener)
 
         public void end()
         {
-            logger.info("Disconnecting " + this.participantId + " from Whisper transcription service.");
-            connectionPool.end(this.roomId, this.participantId);
+            wsClient.disconnectParticipant(this.participantId, allDisconnected -> {});
```

Comment:
Is this supposed to clear the socket from the context when everyone leaves?

Reply:
Yes.